crates/db_schema/src/impls/site.rs

```diff
@@ -65,13 +65,15 @@ impl Site {
   pub async fn read_from_instance_id(
     pool: &mut DbPool<'_>,
     _instance_id: InstanceId,
-  ) -> Result<Option<Self>, Error> {
+  ) -> LemmyResult<Option<Self>> {
```

Comment:
I thought we tried to remove all my `.optional`. Might as well just remove that and return a `NotFound` LemmyResult, instead of wrapping in Ok.